### DISCUSSION TITLE
Volume Settings Design

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#16560] [Ideas]
> So, I decided to make another concept. This time is for _Volume Settings_ style
>
>![default volumes](https://user-images.githubusercontent.com/43073074/45375253-b6e66400-b5f4-11e8-888b-8ba21c2020ae.png)
## Concept
Being able to choose the position (or style) for the _Volume Settings_

## Why?
I think some users prefer choosing the position for the volume (maybe can be "annoying" just on the left side of the screen)

## Ideas (Photoshop)
Here are some ideas:

**A Right-side Design (with the "reflected" bar, cool detail)**

![right design concept](https://user-images.githubusercontent.com/43073074/45375485-64f20e00-b5f5-11e8-9a53-fca7379f84b2.png)

**A Top Design (with the _mute_ button to the bottom)** (but can be reversed: with the _mute_ one on top, and the volumes to the bottom)

![top design concept](https://user-images.githubusercontent.com/43073074/45375547-966ad980-b5f5-11e8-9dab-058a91a22831.png)

**OR,**

instead of choosing the _position_, maybe you can choose the _style_ of it:

**maybe proposing the _old style_ with the new one (like a nostalgia thing)**  (with a _light design_ for volume words, and the _mute_ button in the middle)

![old-style design concept](https://user-images.githubusercontent.com/43073074/45375826-6839c980-b5f6-11e8-861d-9344461527a2.png)

**Comment by peppy** [comment]
In fine with this being added as a setting.

**Comment by GSculerlor** [comment]
Will take this issue, is there any design change to support it? Or is it okay by just align the volume meter based on the position?

**Comment by peppy** [comment]
Just do what looks right and we'll take it from there.

**Comment by peppy** [comment]
@arflyte is the volume control something you've already included in your redesign? i think we want it to be part of the toolbar (but keep in mind it still needs to display as its own floating element when the toolbar is not present).

See #11077 for reasoning.

**Comment by arflyte** [comment]
No changes to the volume control design so far. I'll try to add it to the toolbar.

**Comment by yyny** [comment]
Just putting this here since I don't think this is worth it's own issue, the mute button overlaps other buttons using the bottom left of the screen (like the back button), this is annoying since clicking or pressing ESC doesn't hide the volume menu, meaning you have to wait for the volume menu to disappear before you can press buttons in that part of the screen.
